zipline/pipeline/filters/filter.py

```diff
@@ -488,3 +489,18 @@ def _compute(self, arrays, dates, assets, mask):
                 asset=self._asset, start_date=dates[0], end_date=dates[-1],
             )
         return out
+
+
+class SmoothingFilter(CustomFilter):
```

Comment:
I think we need a more specific name for this.  There are lots of ways we can implement smoothing.  This is one specific way of doing it.